app-ios/.swiftlint.yml

```diff
@@ -41,7 +41,6 @@ opt_in_rules:
 - discouraged_object_literal
 - empty_collection_literal
 - empty_string
-- fallthrough
```

Comment:
Was silencing the rule for `MurMurHash` not an option? Removing an entire rule just for one PR seems a bit extreme.

Reply:
Maybe we could start a poll somewhere to vote on what linting rules to remove. There seems to be a lot of complaints about linting rules.